karton/mwdb_reporter/mwdb_reporter.py

```diff
@@ -71,268 +92,408 @@ def mwdb(self) -> MWDB:
             mwdb.login(mwdb_config["username"], mwdb_config["password"])
         return mwdb
 
-    def _tag_children_blobs(self, config: MWDBConfig) -> None:
-        for item in config.config.values():
-            if type(item) is MWDBBlob:
-                tag = config.family
+    @property
+    def mwdb(self) -> MWDB:
+        if not hasattr(self, "_mwdb"):
```

Comment:
Is there a specific upside to this over just initializing it in the constructor?

Reply:
:thinking: I was a bit afraid of messing with that constructor but actually it sounds better than lazy initatization

Reply:
Done :+1: